src/System.Runtime.Extensions/tests/System/Environment.SetEnvironmentVariable.cs

```diff
@@ -85,9 +83,6 @@ public void Default(EnvironmentVariableTarget target)
 
 
         [Theory]
```

Comment:
You removed all of the inputs from this theory.  The test will no longer run anything.